run-tests.sh

```diff
@@ -2,5 +2,5 @@
 
 cd sonny
 
-IMPORT_CONF=sonny.testing_conf nosetests "$@"
+IMPORT_CONF=sonny.testing_conf nosetests "$@"  --ignore-files="test_temporary_.+\.py"
```

Comment:
Might be good to have a little comment here explaining what we're doing and why. And how to run the DB tests if we want to.